README.md

```diff
@@ -49,7 +49,9 @@ You have two options for authentication depending on where you run Logstash.
 
 1. If you are running Logstash outside of Google Cloud Platform, then you will
 need to create a Google Cloud Platform Service Account and specify the full
-path to the JSON private key file in your config. You must assign sufficient
+path to the JSON private key file in your config in `json_key_file` attribute.
+If you do not want to upload file and want to accept it as a string, then add the content 
+of JSON private key in `json_key_file_content` attribute. You must assign sufficient
```

Comment:
```suggestion
path to the JSON private key file in your config in `json_key_file` setting.
If you don't want to upload the file file and use its content as a string, then add the content 
of JSON private key in `json_key_file_content` setting. You must assign sufficient
```

Reply:
@andsel Thanks for the suggestion I have updated the doc in README and index.asciidoc as per your suggestion.